### PR TITLE
feat(react-components): Event Card

### PR DESCRIPTION
**Pull request by kmaid**

### CODE REVIEW

File: packages/react-components/src/utils.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import { renderToStaticMarkup } from 'react-dom/server';
-import { format } from 'date-fns';
+import { format, utcToZonedTime } from 'date-fns-tz';
```
Comment (kmaid) [inline packages/react-components/src/utils.ts:3]: Not sure we should bother with the polyfil. Its apparently got 95% browser support, thoughts?

File: packages/react-components/src/organisms/EventCard.tsx [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+      >
+        <Anchor href={href}>
+          <Headline4>
+            {title.substr(0, TITLE_LIMIT)}
+            {title.length > TITLE_LIMIT ? '…' : undefined}
+          </Headline4>
+        </Anchor>
+        <div css={dateStyles}>
+          <div>
+            {formatUtcToLocalTimezone(startDate, 'E, d MMM y').toUpperCase()} ∙{' '}
```
[the remark this answers is not in the document]
Comment (jeysal) [reply packages/react-components/src/organisms/EventCard.tsx:91]: Or is there maybe a `anyGivenInputTimezoneToLocalTimezone` that can do the conversion from any input timezone (even if for now it's always utc) to our local timezone?

File: packages/react-components/src/organisms/EventCard.tsx [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+            ? groups.map(({ name, href: groupHref, id }) => (
+                <div css={{ whiteSpace: 'pre' }} key={`group-${id}`}>
+                  <Link href={groupHref}>{name}</Link>{' '}
+                </div>
+              ))
+            : 'ASAP Event'}
+        </span>
+      </div>
+    </div>
+    <TagList tags={tags} max={3} />
```
Comment (jeysal) [inline packages/react-components/src/organisms/EventCard.tsx:112]: Both on mobile and desktop? just confirming

File: packages/react-components/src/organisms/__tests__/EventCard.test.tsx [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+const props: ComponentProps<typeof EventCard> = {
+  ...createEventResponse(),
+  groups: [],
+  href: '',
+};
+
+it('Renders an event', () => {
+  const { getByRole } = render(<EventCard {...props} title="My Event" />);
+  expect(getByRole('heading', { level: 4 })).toHaveTextContent('My Event');
```
Comment (jeysal) [inline packages/react-components/src/organisms/__tests__/EventCard.test.tsx:15]: 4? o.O that's deep, no way it's used that deep down?
I also kinda feel like we should have a heading level react context or something lol

File: packages/react-components/src/organisms/__tests__/EventCard.test.tsx [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+          href: '/my-group',
+        },
+      ]}
+    />,
+  );
+  expect(getByText('My Group')).toHaveAttribute('href', '/my-group');
+  expect(getByTitle('Group')).toBeInTheDocument();
+});
+
+it('shows the the event is run by ASAP when there is no group', () => {
```
Comment (jeysal) [inline packages/react-components/src/organisms/__tests__/EventCard.test.tsx:43]:
```suggestion
it('shows that the event is run by ASAP when there is no group', () => {
```

File: packages/react-components/src/organisms/__tests__/EventCard.test.tsx [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+  expect(getByTitle('Calendar')).toBeInTheDocument();
+});
+
+it('shows that an event has been cancelled', () => {
+  const { getByTitle, getByText } = render(
+    <EventCard {...props} status="Cancelled" />,
+  );
+  expect(getByTitle('Alert')).toBeInTheDocument();
+  expect(getByText(/cancelled/i)).toBeVisible();
+});
```
Comment (jeysal) [inline packages/react-components/src/organisms/__tests__/EventCard.test.tsx:57]: It seems the date and timezone conversion and formatting is the most interesting thing to test here and missing

File: packages/react-components/src/utils.ts (hunk 1 above)
Comment (jeysal) [reply packages/react-components/src/utils.ts:3]: Polyfill what, Intl? nah looks good enough I agree

**Review by jeysal — CHANGES_REQUESTED** [review]

File: packages/react-components/src/molecules/ToastCard.tsx [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+  mobileScreen,
+  tabletScreen,
+  largeDesktopScreen,
+  vminLinearCalc,
+} from '../pixels';
+import { Card, Divider } from '../atoms';
+import { ember } from '../colors';
+import { alertIcon } from '../icons';
+
+const cardPadding = css({
```
[the remark this answers is not in the document]
Comment (kmaid) [reply packages/react-components/src/molecules/ToastCard.tsx:16]: I would have to override half of it

File: packages/react-components/src/organisms/EventCard.tsx (hunk 3 above)
Comment (kmaid) [reply packages/react-components/src/organisms/EventCard.tsx:112]: Its weird on mobile there should be 5 and on desktop there should be three. This passed design review 🤷‍♂️

File: packages/react-components/src/organisms/__tests__/EventCard.test.tsx (hunk 5 above)
Comment (kmaid) [reply packages/react-components/src/organisms/__tests__/EventCard.test.tsx:43]: 😒

**Review by jeysal — APPROVED** [review]

**Pull request merged**